Sources/Models/Files/StoryboardFile.swift

```diff
@@ -19,22 +19,31 @@ public class StoryboardFile: InterfaceBuilderFile {
         self.pathString = path
         self.document = try type(of: self).parseContent(pathString: path)
     }
-
+    
```

Comment:
please do not add new space
review files before committing or use swiftlint autocorrect